packages/react-dom/src/__tests__/ReactDOMImageLoad-test.internal.js

```diff
@@ -0,0 +1,576 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+let React;
+let Scheduler;
+// let ReactCache;
+let ReactDOM;
+// let Suspense;
+let originalCreateElement;
+// let TextResource;
+// let textResourceShouldFail;
+
+let images = [];
+let onLoadSpy = null;
+let actualLoadSpy = null;
+
+function PhaseMarkers({children}) {
+  Scheduler.unstable_yieldValue('render start');
+  React.useLayoutEffect(() => {
+    Scheduler.unstable_yieldValue('last layout');
+  });
+  React.useEffect(() => {
+    Scheduler.unstable_yieldValue('last passive');
+  });
+  return children;
+}
+
+function last(arr) {
+  if (Array.isArray(arr)) {
+    if (arr.length) {
+      return arr[arr.length - 1];
+    }
+    return undefined;
+  }
+  throw new Error('last was passed something that was not an array');
+}
+
+function Text(props) {
+  Scheduler.unstable_yieldValue(props.text);
+  return props.text;
+}
+
+// function AsyncText(props) {
```

Comment:
?

Reply:
I removed (temporarily) the suspense test case at Seb's suggestion

I briefly described the issue above. I'm not yet familiar enough with the Suspense to speak to how this would behave ideally.

>Open Question
>
>We opted out of specifying and asserting a specific behavior on Suspense fallbacks. currently when a load event fires after an img is suspended the event plays out even though the image is part of a suspended tree. We need to figure out what we want the event replaying behavior to be here

Reply:
xit gets linted against since it’s something you do temporarily and disabling the lint makes a similar accidental point.

Since xit is a temporary measure that you use while developing I feel like it’s the wrong primitive for checking in disabled code.